packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -350,6 +352,32 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
     });
   };
 
+  getAllSelectableChildrenId = items => {
+    return items.reduce((acc, item) => {
+      if (item.items) {
+        return [...acc, ...this.getAllSelectableChildrenId(item.items)];
+      }
+      return item.hasOwnProperty('selectable') && !item.item ? acc : [...acc, item.id];
```

Comment:
```suggestion
      return item.hasOwnProperty('selectable') && !item.items ? acc : [...acc, item.id];
```
Shouldn't this be `item.items`

Reply:
The correct should be `item.hasOwnProperty('selectable') && !item.selectable`